filemanager/filemanager.go

```diff
@@ -74,6 +74,11 @@ type Settings struct {
 	Logger   logger.Logger
 	Conf     *config.Config
 
+	// when S3ManagerV2 is set to true, the client uses the new S3 manager
+	S3ManagerV2 bool
+
+	// when DigitalOceanSpacesManagerV2 is set to true, the client uses the new DigitalOcean Spaces manager
+	DigitalOceanSpacesManagerV2 bool
```

Comment:
These feel a bit out of place here (same applies to the `GCSUploadIfNotExist` field). We shouldn't have provider-specific fields in a generic settings struct. Can we move them in `Config`?

btw, I couldn't find where we're using `GCSUploadIfNotExist`

Reply:
The only place I could find it being used is in a test. Removing this